app.py

```diff
@@ -1,12 +1,36 @@
 #!/usr/bin/env python
-from flask import Flask, render_template, Response, request
-
+from flask import Flask, render_template, Response, request, flash
+from flask_sqlalchemy import SQLAlchemy
 import mediapipe_processor
+import psycopg2
 
 Camera = mediapipe_processor.Camera
 
 app = Flask(__name__)
 
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:8543abcd@localhost/ElecTrap_scoreboard'
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.secret_key = 'secret string'
+
+db = SQLAlchemy(app)
+
+con = psycopg2.connect(database="ElecTrap_scoreboard", user="postgres", password="8543abcd", host="127.0.0.1", port="5432")
+cursor = con.cursor()
+
+class userinfo(db.Model):
+    user_id = db.Column(db.Integer, primary_key =True)
+    user_name = db.Column(db.String(20))
+    game_mode = db.Column(db.String(20))
+    game_body = db.Column(db.String(20))
+    game_level = db.Column(db.Integer)
+    score = db.Column(db.Integer)
+
+    def __init__(self,user_name, game_mode, game_body, game_level, score):
+        self.user_name = user_name
+        self.game_mode = game_mode
+        self.game_body = game_body
+        self.game_level = game_level
+        self.score = score 
```

Comment:
Model class should be put in another file like `model.py` to follow the MVC convention.